sources/install/package_crypto.sh

```diff
@@ -3,6 +3,20 @@
 
 source common.sh
 
+function install_crypto_apt_tools() {
+    # CODE-CHECK-WHITELIST=add-aliases
+    colorecho "Installing crypto apt tools"
+
+    fapt sagemath
+
+    add-history sage
```

Comment:
You need to add history commands please.

Reply:
On the line 12, i see `add-history sage`, so it's good, no ?

Reply:
If you add : add-history, you need to create a file containing history commands.

example : https://github.com/ThePorgs/Exegol-images/blob/main/sources/assets/shells/history.d/azure-cli